source/create-agg-pipeline.txt

```diff
@@ -318,7 +317,7 @@ your chosen view mode:
       .. procedure::
          :style: normal
          
-         .. step:: Select the :guilabel:`Text` view.
+         .. step:: Select the Text view
```

Comment:
the one agg "task" that might not be represented here is importing a pipeline that's already been written. we used to have an explicit menu option to allow users to do this, now we just expect them to copy-paste into the agg text editor. do you think this should be its own task page? we do periodically get questions about this

Reply:
I think because the menu option was removed, it seemed like the directions here were almost exactly the same as those written to explain how to create a pipeline on Text View Mode. I can add an admonition or update the text under the Text View Mode section of "Create an Aggregation Pipeline" that mentions this!